### PR TITLE
[WIKI] Why not x86

### PR DESCRIPTION
**Pull request by uilianries**
Hi!

As it's not possible to create a PR to the Wiki, and we don't have a Docs/ folder yet, I gonna write this PR to get reviews for a new FAQ in the Wiki.

/cc @Croydon

### CODE REVIEW

File: docs/faq/why_not_x86.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+* Each recipe results around 130 packages, and this is only for x86_64, but not all packages are used, some settings remain with zero downloads throughout their life. So, imagine adding more settings that will be rarely used, but that will consume more resources as time and storage, this leaves us in an impractical situation.
+
+#### But if there are no packages available, what will the x86 validation look like?
+
+As stated earlier, any increase in the number of configurations will result in an impractical scenario. In addition, more validations require more review time for a recipe, which would increase the time for all PRs, delaying the release of a new package. For these reasons, x86 is not validated by the CCI.
+
+We often receive new fixes and improvements to the recipes already available for x86_64, including help for other architectures like x86 and ARM. In addition, we also receive new cases of bugs, for recipes that do not work on a certain platform, but that are necessary for use, which is important to understand when we should put more effort. So we believe that the best way to maintain and add support for other architectures is through the community.
```
Comment (Croydon) [inline docs/faq/why_not_x86.md:14]:
```suggestion
We often receive new fixes and improvements to the recipes already available for x86_64, including help for other architectures like x86 and ARM. In addition, we also receive new cases of bugs, for recipes that do not work on a certain platform, but that are necessary for use, which is important to understand where we should put more effort. So we believe that the best way to maintain and add support for other architectures is through the community.
```

`when` -> `where`?

File: docs/faq/why_not_x86.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+## Why not x86 binaries?
+
+As described in the [Supported platforms and configurations](https://github.com/conan-io/conan-center-index/wiki/Supported-Platforms-And-Configurations), only the x86_64 architecture is available for download, the rest must be built from sources. The reasons behind this decision are:
+
+* Few users need different pre-built packages that are either x86_64, this number is less than 10% of total users (data obtained through the download counter from Bintray), and tends to decrease over the years;
```
Comment (Croydon) [inline docs/faq/why_not_x86.md:5]:
```suggestion
* Few users need different pre-built packages that are not x86_64 packages, this number is less than 10% of total users (data obtained through the download counter from Bintray), and tends to decrease over the years;
```

File: docs/faq/why_not_x86.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+* Some OS are putting the x86 as obsolete, examples [macOS](https://developer.apple.com/documentation/macos-release-notes/macos-catalina-10_15-release-notes) and Ubuntu 20.04;
+* For security reasons, most companies build their own packages from sources, even if they already have a pre-built version available, which further reduces the need for extra configurations;
+* Each recipe results around 130 packages, and this is only for x86_64, but not all packages are used, some settings remain with zero downloads throughout their life. So, imagine adding more settings that will be rarely used, but that will consume more resources as time and storage, this leaves us in an impractical situation.
```
Comment (SSE4) [inline docs/faq/why_not_x86.md:8]:
```suggestion
* Each recipe results in around 130 packages, and this is only for x86_64, but not all packages are used, some settings remain with zero downloads throughout their life. So, imagine adding more settings that will be rarely used, but that will consume more resources as time and storage, this leaves us in an impractical situation.
```

**Comment by danimtb** [comment]
I think we can start by having just a *faq.md* with all the questions together. Later we can create subsections if there are questions related to the same topic. WDYT?

**Comment by uilianries** [comment]
Sounds good. I would prefer docs/faqs.md, but as you said, we can provide it in the future.

**Comment by danimtb** [comment]
@uilianries yes, I meant *docs/faws.md* 💯

**Comment by uilianries** [comment]
@danimtb  Done!

**Pull request merged**